db/mongo.go

```diff
@@ -0,0 +1,87 @@
+package db
+
+import (
+	"log"
+
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+//Technology collects the slack user ID of people who use or want to learn a
+//given technology, identified by Name
+type Technology struct {
+	ID       string   `json:"id"`
+	Name     string   `json:"name"`
+	Users    []string `json:"users"`
+	Learners []string `json:"learners"`
+}
+
+func getSession() *mgo.Session {
+	session, err := mgo.Dial("mongodb://localhost")
```

Comment:
Do you want the overhead of redialing every time? You could create the initial session once and then copy or clone it.